### PR TITLE
159 refactor profile front

### PR DESCRIPTION
**Pull request by Ycaro02**

### CODE REVIEW

**Review by Pixailz — APPROVED** [review]
Good job, working great and the nickname spin well :p

**Review by Pixailz — APPROVED** [review]
noice, working great :)

File: docker-compose-dev.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ version: "3.9"
 # DEFAULT CONF https://stackoverflow.com/a/65522511
 
 x-healthcheck_conf: &healthcheck_conf
-      interval: 5s
```
Comment (Pixailz) [inline docker-compose-dev.yaml:7]: Why ?

this change a lot on the CI side, 5min is good enought to deploy the whole project

**Review by Pixailz — CHANGES_REQUESTED** [review]
Some change have to be done according to comment

**Review by RuBeeN-cmd — APPROVED** [review]
Nice work

**Review by RuBeeN-cmd — APPROVED** [review]
Resolved

**Review by Pixailz — APPROVED** [review]
Noice :heart:

**Comment by Pixailz** [comment]
don't forget to squash before merging, noice improvement on the artistic side :D

**Pull request merged**